test/TestUtilities/Test.Utility/Commands/ProjectTestHelpers.cs

```diff
@@ -357,6 +357,26 @@ public static PackageSpec GetPackageSpec(string projectName, string rootPath, st
             return GetPackageSpecWithProjectNameAndSpec(projectName, rootPath, spec);
         }
 
+        public static PackageSpec GetPackageSpecWithFloatingVersion(string projectName, string rootPath, string framework, string dependencyName, string dependencyVersion, bool useAssetTargetFallback = false, string assetTargetFallbackFrameworks = "", bool asAssetTargetFallback = true)
```

Comment:
I think the name can/should remain GetPackageSpec as the input determines where this is a package spec with a floating version of not. The method isn't doesn't do it.